app/src/main/java/com/blessapp/blessapp/Admin/AdminNewOrdersActivity.java

```diff
@@ -1,16 +1,137 @@
+                .setQuery(ordersRef, Orders.class)
+                .build();
+
+        FirebaseRecyclerAdapter<Orders, OrdersViewHolder> adapter =
+                new FirebaseRecyclerAdapter<Orders, OrdersViewHolder>(options) {
+                    @Override
+                    protected void onBindViewHolder(@NonNull OrdersViewHolder holder, final int position, @NonNull Orders model) {
+                        //To make sure they display the decimal places
+                        totalPrice = Float.valueOf(model.getTotalAmount());
+                        userNameAsKey = model.getUsername();
+
+                        holder.name.setText("Name : " + model.getName());
+                        holder.phone.setText("Phone Number : " + model.getPhone());
+                        holder.totalPrice.setText("Total : RM " + df.format(totalPrice));
+                        holder.dateTime.setText("Date & Time Ordered : " + model.getDate() + ", " + model.getTime());
+
+                        holder.details.setOnClickListener(new View.OnClickListener() {
+                            @Override
+                            public void onClick(View v) {
+                                String uID =  getRef(position).getKey();
+
+                                Intent intent= new Intent(AdminNewOrdersActivity.this, AdminOrderDetailsActivity.class);
+                                intent.putExtra("Order Primary Key", uID);
+                                startActivity(intent);
+                            }
+                        });
+
+                        holder.delivery.setOnClickListener(new View.OnClickListener() {
+                            @Override
+                            public void onClick(View view) {
+                                CharSequence options[] = new CharSequence[]{
+                                        "Yes", "No"
+                                };
+
+                                AlertDialog.Builder builder = new AlertDialog.Builder(AdminNewOrdersActivity.this);
+                                builder.setTitle("Packed and ready to be deliver?");
+
+                                builder.setItems(options, new DialogInterface.OnClickListener() {
+                                    @Override
+                                    public void onClick(DialogInterface dialog, int which) {
+                                        if (which == 0){
+                                            //We want to display what we actually click, so:
+                                            //getRef(position) = to use position as reference to pass to another activity
+                                            //getKey() = to get key from database, so that the next activity will display data accurately instead of mixing with other user's order
+                                            String uID =  getRef(position).getKey();
+
+                                            RemoveOrder(uID, userNameAsKey);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.